worker/task.go

```diff
@@ -1912,6 +1912,14 @@ func parseSrcFn(ctx context.Context, q *pb.Query) (*functionContext, error) {
 			}
 		}
 
+		generateIneqTokens := true
+		if fc.fname != eq && q.UidList != nil && uint64(len(q.UidList.Uids)) < Config.TypeFilterUidLimit {
+			if !checkUidZero(q.UidList.Uids) {
+				fc.n = len(q.UidList.Uids)
+				generateIneqTokens = false
```

Comment:
nit: should add a comment here as to why we are choosing to set generateIneqTokens=false